package p1;

import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class Demo {

	public static void main(String[] args) {
		Scanner kb = new Scanner(System.in);
		File file = new File("data.txt");
		try {
			Scanner in = new Scanner(file);
			String input = in.nextLine();
			System.out.println(input);
		} catch (FileNotFoundException e) {
			System.out.println("File not found, enter a new name: ");
		}
		
		
		
//		int[] numbers = {0, 1, 2, 3, 4, 5};
//		
//		try {
//			System.out.println(numbers[6]); // unchecked exception
//			System.out.println("after exception");
//		} catch (Exception ex) {
//			System.out.println("hhahaa some error happened.");
//		}
//		System.out.println("finished");
//		
//		// checked exception
//		PrintWriter pw;
//		try {
//			pw = new PrintWriter("data.txt");
//			pw.write("hi");
//			pw.close();
//		} catch (FileNotFoundException e) {
//			e.printStackTrace();
//		}
//		System.out.println("Whatever in here will still work when there is try-catch");
//		
		
	}

}
